feat(changelog): object-storage-added-bucket-policies-can-now-be-configu 2024-07-02 (#3424)
scaleway/docs-content: changelog/july2024/2024-07-02-object-storage-added-bucket-policies-can-now-be-configu.mdx

---

> title: Bucket policies can now be managed from the console!

Reviewer: ```suggestion
title: Bucket policies can now be managed from the Scaleway console!
```

> title: Bucket policies can now be configured from the Console!

Reviewer: ```suggestion
title: Bucket policies can now be managed from the Console!
```

> title: Bucket policies can now be configured from the Console!

Reviewer 2: ```suggestion
title: Bucket policies can now be configured from the Scaleway console!
```

status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-07-02
category: storage
product: object-storage
---

> Bucket policies can now be created, viewed and edited from the Console next to your buckets list. Additionally, you can now see which buckets lack policies and get a centralized view of all your bucket policies. 
> 
> Simple and intuitive access management is key for reducing the risk of configuration errors and enhancing security. With a user-friendly interface that allows to both configure IAM and Bucket policies, granting granular access to your Object Storage ressources is easier and more reliable than ever. 
> 
> Learn here https://www.scaleway.com/en/docs/storage/object/how-to/create-bucket-policy/ how to create and manage your bucket policies from the Console.

Reviewer: ```suggestion
You can now create, view and edit bucket policies directly from the Scaleway console, right next to your buckets list. Additionally, you can now see which buckets lack policies and get a centralized view of all your bucket policies. 

Simple and intuitive access management is key for reducing the risk of configuration errors and enhancing security. With a user-friendly interface that allows you to configure both IAM and bucket policies, [granting granular access to your Object Storage resources](/storage/object/api-cli/combining-iam-and-object-storage/) is much easier, and more reliable than ever. 

Refer to our [dedicated documentation](/storage/object/how-to/create-bucket-policy/] to discover how to create and manage your bucket policies from the [Scaleway console](http://console.scaleway.com).
```

> You can now create, view and edit bucket policies directly from the Scaleway console, right next to your buckets list. Additionally, you can now see which buckets lack policies and get a centralized view of all your bucket policies. 
> 
> Simple and intuitive access management is key for reducing the risk of configuration errors and enhancing security. With a user-friendly interface that allows you to configure both IAM and bucket policies, [granting granular access to your Object Storage resources](/storage/object/api-cli/combining-iam-and-object-storage/) is much easier, and more reliable than ever. 
> 
> Refer to our [dedicated documentation](/storage/object/how-to/create-bucket-policy/] on how to create and manage your bucket policies from the [Scaleway console](http://console.scaleway.com) for more information.

Reviewer: ```suggestion
Bucket policies can now be created, viewed and edited from the [Scaleway console](https://console.scaleway.com/). You can now see which buckets lack a policy, and get a centralized view of all your bucket policies. 
Simple and intuitive access management reduces configuration errors and enhances security. Granting granular access to your Object Storage resources has never been easier with our user-friendly interface that allows you to configure both IAM and bucket policies.
Find out how [create and manage your bucket policies](/storage/object/how-to/create-bucket-policy/) from the Scaleway console.
```
